### DISCUSSION TITLE
Search microservice

### DISCUSSION THREAD

**Discussion by MadalenaDiasSec** [LesterCerioli/StockControl#6] [General]
I intend to build a microservice that will provide search resources for everyone microservices.

Important points to consider.
- [x] The microservice will allow you to find search results for registered products and their appropriate categories
- [x] provide surveys of user records, brands, suppliers, document data

**Comment by LesterCerioli** [comment]
Very great @MadalenaDiasSec  We need use Dapper to read queries and EF Core ot recird queries and it it we get provide more performance for our services.
